src/lib/state/db/dbOperations.ts

```diff
@@ -4,6 +4,15 @@ const storeName = "stateStore"
 const dbName = "UplinkAppState"
 
 export async function initDB(): Promise<IDBDatabase> {
+    // During e.g. a page refresh initDB() is called too fast and the constants are uninitialized. This is a workaround for that
```

Comment:
Sounds like a race is created? In that case, I wonder if the overall initialization should be restructured, but could be left to a different discussion.